cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/ObjectMappedTableProperties.java

```diff
@@ -137,7 +137,8 @@ public Builder setRowKeyExploreName(String name) {
     /**
      * Sets the column type for the row key column in the corresponding Hive table for an ObjectMappedTable.
      * By default, the type of the row key in your Hive table will be binary. You can set the type using this
-     * method. Only {@link Schema.Type#BYTES} and {@link Schema.Type#STRING} are allowed.
+     * method. Only {@link co.cask.cdap.api.data.schema.Schema.Type#BYTES Schema.Type.BYTES} and
```

Comment:
I don't know. It just doesn't work without it. Perhaps because it is part of an imported class. Would the complier find it?